pkg/Analisis.go

```diff
@@ -0,0 +1,7 @@
+package pkg
+
+type Analisis struct{
+	Prediccion string
```

Comment:
No tratéis de representar como atributos absolutamente todo. `Prediccion` para empezar, no corresponde a este milestone. Pero lo importante es que se trataría del resultado de una función.